config/database.yml

```diff
@@ -0,0 +1,89 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On macOS with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On macOS with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: Blog_App_development
+  user: postgres
+  password: Utnubb@83
```

Comment:
- [ ] [ OPTIONAL] Never use the password directly, always use the environment variable to represent it & mention that environment file in the `.gitignore` file. [Read here ](http://railsapps.github.io/rails-environment-variables.html)how to implement  it.

Reply:
Thank you for this, I have been wondering how can I do that.